### PR TITLE
AAU, on playground, when I query assets, I see I can query the assets metadata

### PR DESCRIPTION
**Pull request by p-desaintchamas**
- [x] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests

### CODE REVIEW

File: kili/helpers.py [hunk 1]
```diff
@@ -90,12 +90,16 @@ def format_json(result):
     if isinstance(result, dict):
         for key, value in result.items():
             if key in ['jsonInterface', 'jsonMetadata', 'jsonResponse']:
-                if value == '' or value is None:
+                if key == 'jsonMetadata' and is_url(result[key]):
+                    print('escape')
+                    result[key] = result[key]
```
Comment (marcenacp) [inline kili/helpers.py:95]: First part of the `if` could be skipped?

File: kili/helpers.py [hunk 2]
```diff
@@ -90,12 +90,16 @@ def format_json(result):
     if isinstance(result, dict):
         for key, value in result.items():
             if key in ['jsonInterface', 'jsonMetadata', 'jsonResponse']:
-                if value == '' or value is None:
+                if key == 'jsonMetadata' and is_url(result[key]):
+                    print('escape')
```
Comment (marcenacp) [inline kili/helpers.py:94]: Print

File: kili/helpers.py [hunk 3]
```diff
@@ -90,12 +90,16 @@ def format_json(result):
...
+                    result[key] = result[key]
+                elif value == '' or value is None:
                     result[key] = dict()
                 else:
                     try:
                         result[key] = loads(value)
                     except:
+                        print(key, value)
```
Comment (marcenacp) [inline kili/helpers.py:102]: Print

File: kili/helpers.py (hunk 1 above)
Comment (p-desaintchamas) [reply kili/helpers.py:95]: What do you mean ? We can do only the check is_url(value) ?

**Review by marcenacp — APPROVED** [review]

**Pull request merged**